deployment/CP05MOAS-GL340_Deploy.csv

```diff
@@ -41,9 +41,9 @@ P8a,,AR-24,,CP05MOAS-GL340-03-CTDGVM000,7,1,2017-08-17T08:20:00,2017-10-28T19:16
 P8a,,AR-24,,CP05MOAS-GL340-04-DOSTAM000,7,1,2017-08-17T08:20:00,2017-10-28T19:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00123,40.00702,-70.84823,,200,200,EB 
 P8a,,AR-24,,CP05MOAS-GL340-05-PARADM000,7,1,2017-08-17T08:20:00,2017-10-28T19:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50252,40.00702,-70.84823,,200,200,EB 
 P8a,,AR-24,,CP05MOAS-GL340-01-ADCPAM000,7,1,2017-08-17T08:20:00,2017-10-28T19:16:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-00541,40.00702,-70.84823,,200,200,EB 
-TS20190205,,,,CP05MOAS-GL340-00-ENG000000,8,1,2019-02-05T08:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,39.91563333,-70.36722222,,200,200,EB 
-TS20190205,,,,CP05MOAS-GL340-02-FLORTM000,8,1,2019-02-05T08:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-03884,39.91563333,-70.36722222,,200,200,EB 
-TS20190205,,,,CP05MOAS-GL340-03-CTDGVM000,8,1,2019-02-05T08:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09055,39.91563333,-70.36722222,,200,200,EB 
-TS20190205,,,,CP05MOAS-GL340-04-DOSTAM000,8,1,2019-02-05T08:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00436,39.91563333,-70.36722222,,200,200,EB 
-TS20190205,,,,CP05MOAS-GL340-05-PARADM000,8,1,2019-02-05T08:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50153,39.91563333,-70.36722222,,200,200,EB 
-TS20190205,,,,CP05MOAS-GL340-01-ADCPAM000,8,1,2019-02-05T08:24:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-00537,39.91563333,-70.36722222,,200,200,EB 
\ No newline at end of file
+TS20190205,,BK20190504,,CP05MOAS-GL340-00-ENG000000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,39.91563333,-70.36722222,,200,200,EB 
+TS20190205,,BK20190504,,CP05MOAS-GL340-02-FLORTM000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-03884,39.91563333,-70.36722222,,200,200,EB 
+TS20190205,,BK20190504,,CP05MOAS-GL340-03-CTDGVM000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09055,39.91563333,-70.36722222,,200,200,EB 
+TS20190205,,BK20190504,,CP05MOAS-GL340-04-DOSTAM000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00436,39.91563333,-70.36722222,,200,200,EB 
+TS20190205,,BK20190504,,CP05MOAS-GL340-05-PARADM000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50153,39.91563333,-70.36722222,,200,200,EB 
+TS20190205,,BK20190504,,CP05MOAS-GL340-01-ADCPAM000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-00537,39.91563333,-70.36722222,,200,200,EB 
```

Comment:
Reviewed by RT - looks good.

Reply:
It looks like exact time of recovery is not known.  It would be good to note this on line 44 in the notes column.  Otherwise this looks fine.